spec/dummy/db/seeds.rb

```diff
@@ -0,0 +1,5 @@
+Caffeine::AdminUser.create(email: 'admin@example.com', password: 'q1w2e3')
+
+Caffeine::Page.create(title: 'main', position: 0, data: {content: '', summary: '', serve_by_slug_only: 0}, main: true)
+Caffeine::Page.create(title: 'parent', position: 1, data: {content: '', summary: '', serve_by_slug_only: 0}, main: true)
+Caffeine::Page.create(title: 'child', position: 2, data: {content: '', summary: '', serve_by_slug_only: 0}, main: true)
```

Comment:
Line is too long. [119/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.